mobile/src/people/avatar.tsx

```diff
@@ -54,12 +53,12 @@ function validateEncodedImage(data: string) {
 }
 
 export class Avatar extends Component<AvatarProps, AvatarState> {
-    constructor(props: AvatarProps) {
-        super(props);
-        this.state = {};
-    }
+    public state: AvatarState = {
+        size: NaN,
+        borderRadius: NaN
+    };
 
-    public onLayout = (e: LayoutChangeEvent) => {
+    public onLayout = (e: any) => {
```

Comment:
?

Reply:
I'll revert it to newest version of the component - this one was used as a quick fix for the issue with incorrect size of photo (for some reason better coded and styled component drawing incorrect Image) - I'll turn it back and try to figure out why it's happening.